### ISSUE TITLE
Imputer Taking too long??

### ISSUE THREAD

**Issue by AnishNehete** [alpercakr/Planet-Hunting-and-Habitability-Detection-with-Machine-Learning#2]
I fired up the code cell which contained the imputer and even after 300 mins the Imputed Data Hasn't been copied on to the CSV file??

**Comment by alpercakr** [comment]
Hi,
Did you use the code that I wrote? made any changes?

**Comment by AnishNehete** [comment]
Yes,I forked the code and made no changes.
I tried with random estimators like Bayesian Ridge, Decision trees,KNN and all of them gave an imputed data CSV file after 10 mins of Execution. The same did not happen with Random Forest Regressor.
P.S. - It would be better if you added the exoplanet CSV file in the git, as the newer CSV files of the planetary systems contains 315 columns instead of the 355 columns which would lead to some more errors later on in the code. I know this is an old project but still, it would be of great help!

**Comment by alpercakr** [comment]
Yes, you are right, but unfortunately, I couldn't find the data.

**Comment by AnishNehete** [comment]
Never mind, Thanks for replying, I still learned a lot from your project.

**Comment by alpercakr** [comment]
It's good to hear that. Have a good day.

**Comment by AnishNehete** [comment]
Is there anyway I can use the new data and then use the imputer and increase the accuracy of the model?
